streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/worker/context/RayRuntimeContext.java

```diff
@@ -16,6 +27,15 @@
   private Long batchId;
   private final Long maxBatch;
   private Map<String, String> config;
+  /**
+   * Backend for keyed state. This might be empty if we're not on a keyed stream.
+   */
+  protected transient KeyStateBackend keyStateManager;
```

Comment:
please follow the standard java naming convention, use ***Backend instead of ***Manager